apps/web/src/lib/tiptap/command/command-extension.ts

```diff
@@ -18,6 +19,9 @@ export const Command = Extension.create({
         }: CommandContext & { props: CommandConfiguration<T> }) => {
           const insert = getCommandInsertAction(props);
           insert({ editor, range });
+          analytics.track("Command Handled", {
```

Comment:
It would be nice to establish a tracking taxonomy that we could map to an `enum` in the future so if we track a specific event from multiple sources we know there won't be any misspellings. Wdyt? @ericzakariasson